starter-spring/src/test/java/io/micrometer/prometheus/rsocket/autoconfigure/PrometheusRSocketClientAutoConfigurationTest.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2019 Pivotal Software, Inc.
+ * Copyright 20190 Pivotal Software, Inc.
```

Comment:
What calendar are you using? :)

Reply:
Bahaha - its next gen stuff. ;) 

I went to update copyrights in the files I touched but noticed all of the files were at `2019` so figured the copyright requirements in this repo were just using the base inception date. I missed this "undo" though. 

Lmk if I need to update them.